templates/languages/golang/providers/mise/Makefile

```
@@ -0,0 +1,70 @@
+LINT_MODE ?= fix
+BUILD_DIR := build
+MISE ?= $(CURDIR)/.provider/bin/mise
+RUN := $(MISE) exec --
+
+help: ## Show available make targets
+	@echo "{{REPOSITORY_NAME}} — Available Commands (mise):"
+	@echo ""
+	@grep -hE '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | \
+		awk 'BEGIN {FS = ":.*?## "}; {printf "  \033[36m%-18s\033[0m %s\n", $$1, $$2}'
+
+setup-env: ## Setup isolated environment with mise
```

checkmake (0.3.2)

[warning] Target body for "setup-env" exceeds allowed length of 5 lines (12).

(maxbodylength)